gtk4/src/expression.rs

```diff
@@ -182,6 +182,56 @@ impl glib::value::ToValueOptional for Expression {
     }
 }
 
+/// Trait containing convenience methods in creating [`gtk::PropertyExpression`] that
```

Comment:
all the docs here should be prefixed with rustdoc-stripper-ignore-next comment; look for it on other codes ;-)

Reply:
Done, i think :D